examples/dash/dash-connect-pgsql/data/csv_mod.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+csv_mods = {
+    "student-mat.csv": "student-mat-min.csv",
+    "student-por.csv": "student-por-min.csv"
+}
+
+for csv_cur, csv_conv in csv_mods.items():
+    df = pd.read_csv(csv_cur, sep=";")
+    #print(df.columns)
+    df = df[["school", "sex", "romantic_status", "age", 
+        "mother_occupation", "father_occupation", 
+            "health", "study_time", "absences", "final_grade"]]
+    print(df.head())
+    df.to_csv(csv_conv)
```

Comment:
just remove the 2 print statements please

Reply:
also since you added this file, please mention it in the blog when you discuss setting up the data

Reply:
[Removed](https://github.com/ploomber/doc/pull/259/commits/68f43337a32dfdf0f00ffdd974b3cd7995f2ef36#diff-8021001209dc386f683f2e962a504491e43c1b83bb717cd644d5e98d4eb5d79eR10-R15)